medicines/doc-index-updater/Makefile

```diff
@@ -32,13 +32,12 @@ ci-branch: docker-build ## Build branch in CI
 ci-master: ci-branch docker-push ## Build master in CI
 
 .PHONY: docker-build
-docker-build: ## Build and tag Docker image
-	docker build \
+docker-build: ## Build and tag Docker image (with search-client create as dependency)
+	tar cv --exclude=target Dockerfile -C .. ./doc-index-updater ./search-client | docker build - \
```

Comment:
Would it perhaps be simpler just to move the docker file up one folder (and maybe rename it to something like `Dockerfile.doc-index-updater`) so then you could just do `docker build -f Dockerfile.doc-index-updater`?

Reply:
We did consider that but went with this so that the Dockerfile for doc-index-updater is actually with it - could get messy in the folder above if more appear over time.